Bottom: 95460fe5374d44c039db936540a7cc68bbf1716c
Top:    962c3963f77fff9b6a558af92dc2e27c50738004
Author: Edward "Teddy" Brow <edward.teddy.brow@gmail.com>
Date:   2017-04-21 16:27:14 -0700

Refresh of 1.14

---

diff --git a/01_14/notes.txt b/01_14/notes.txt
new file mode 100644
index 0000000..eba7e2e
--- /dev/null
+++ b/01_14/notes.txt
@@ -0,0 +1,4 @@
+for loops are most useful when one needs to keep track of some counter or number that is modified in a consistent way
+each iteration through a loop and/or you can predict or compute how many iterations are needed. A while loop, however,
+is more appropriate when one wants to wait to terminate until some condition is met, but it's not possible or not
+straightforward to compute when that will come to pass.
